StripeConnect/StripeConnect/Source/Internal/Analytics/ComponentAnalyticsClient.swift

```diff
@@ -240,23 +241,15 @@ class ComponentAnalyticsClient {
 extension ComponentAnalyticsClient.CommonFields {
     init(apiClient: STPAPIClient,
          component: ComponentType,
+         publicKeyOverride: String?,
          componentInstance: UUID = .init()
     ) {
```

Comment:
optional - what do you think about passing the `ConnectJSURLParams` directly in `init` instead of separate `apiClient`, `component`, and `publicKeyOverride` params? We already have to construct the URL params in the ComponentManager, so we could just do that earlier in the stack and forward it here.